style-about.css

```diff
@@ -0,0 +1,535 @@
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-bottom: 2rem;
+  }
+
+  .chef-event-1 .layer {
+    background-color: rgba(226, 110, 14, 0.692);
+    z-index: 10;
+    background-position: center;
+    width: 37vh;
+    height: 25vh;
+  }
+
+  .chef-event-2 {
+    background-image: url("./img/event-2.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .chef-event-2 .layer {
+    background-color: rgba(226, 110, 14, 0.692);
+    z-index: 10;
+    background-position: center;
+    width: 37vh;
+    height: 25vh;
+  }
+
+  .layer .year {
+    font-family: lato, sans-serif;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: baseline;
+    padding-top: 4.5rem;
+    color: white;
+  }
+
+  .layer p {
+    display: flex;
+    justify-content: center;
+    color: white;
+  }
+
+  .partner {
+    background-color: #270404;
+    color: white;
+  }
+
+  .partner h3 {
+    text-align: center;
+    padding-top: 2rem;
+    text-decoration: underline;
+    text-decoration-color: #c7724b;
+    text-underline-offset: 0.6rem;
+    text-decoration-thickness: 10%;
+  }
+
+  .partner-list li {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 1.5rem;
+  }
+
+  footer .footer-list p {
+    color: black;
+  }
+
+  .partner-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    margin-top: 1rem;
+    list-style: none;
+  }
+
+  .img-fluid {
+    max-width: 45%;
+  }
+
+  .footer-list {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: 50% 50%;
+    justify-content: space-around;
+  }
+
+  .copy {
+    grid-column-start: 1;
+    display: flex;
+    justify-content: center;
+    margin-top: 2rem;
+  }
+
+  .peru {
+    margin-top: 2rem;
+  }
+
+  .rights {
+    grid-column-start: 2;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .openMenu,
+  .closeMenu {
+    display: none;
+  }
+
+  nav {
+    display: grid;
+  }
+
+  nav ul {
+    background-color: white;
+    display: flex;
+    gap: 2rem;
+    justify-content: space-around;
+    list-style: none;
+    padding-top: 1.2rem;
+    padding-bottom: 1.2rem;
+    align-items: center;
+    border-bottom: 1px solid rgba(49, 49, 49, 0.247);
+  }
+
+  ul a {
+    text-decoration: none;
+    color: black;
+  }
+
+  .top-bar {
+    display: flex;
+    gap: 2rem;
+    justify-content: flex-end;
+    background-color: #8a7f7a;
+    color: white;
+    padding: 0.5rem;
+    padding-right: 5rem;
+  }
+
+  .chef-icon {
+    max-width: 60%;
+  }
+
+  .nav-4 {
+    border: 3px solid #c7724b;
+    padding: 0.6rem;
+    color: #c7724b;
+  }
+
+  header {
+    background-image: url("./img/background-h1.jpg");
+  }
+
+  .first-page {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: repeat(6, 1fr);
```

Comment:
- [ ] Right now, the empty grid-rows are taking too much space, to make it match the original design better, as requested by the previous CR, you should consider making those empty spaces a bit smaller.